src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -544,14 +557,15 @@ void validateJobLabelKeyValue(FailureCollector failureCollector) {
           !containsMacro(NAME_INITIAL_RETRY_DURATION) && !containsMacro(NAME_MAX_RETRY_DURATION) &&
           !containsMacro(NAME_MAX_RETRY_COUNT) && !containsMacro(NAME_RETRY_MULTIPLIER)) {
           validateRetryConfiguration(
-            failureCollector, initialRetryDuration, maxRetryDuration, maxRetryCount, retryMultiplier
+            failureCollector, initialRetryDuration, maxRetryDuration, maxRetryCount, retryMultiplier, readTimeout
```

Comment:
I think here `readTimeout` should be provided using the `getter`: `getReadTimeout`, otherwise the validation will fail for existing pipelines which will have `readTimeout` as `null`.